src/components/common/NameRequestInfo.vue

```diff
@@ -176,58 +163,9 @@ export default class NameRequestInfo extends Vue {
   }
 
   /** The applicant. */
-  get applicant (): NameRequestApplicantIF {
-    return this.getNameRequest.applicants // not an array
-  }
-
-  /** The applicant's name. */
-  get applicantName (): string {
```

Comment:
Now using a common (soon to be shared) mixin method for this.

Ditto for applicant address below.

Reply:
We can look at making the common mixin shared later.